FND/Agreements/Contracts.rdf

```diff
@@ -356,6 +386,38 @@
 		<cmns-av:synonym>master contract</cmns-av:synonym>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-agr-ctr;MilestoneEvent">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-oc;Occurrence"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&cmns-cls;isClassifiedBy"/>
+				<owl:onClass rdf:resource="&fibo-fnd-agr-ctr;ContractMilestone"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;exemplifies"/>
```

Comment:
This is the original property in the ontology that defines an occurrence. And there are actually two restrictions - this one, and one saying that it is classified by the occurrence kind. I'll remove the exemplifies restriction, since we have the other isClassifiedBy one, and then we should discuss raising an issue against the schedule if you would like to use a different name for that property (if we need it at all).